security/advancedtls/sni.go

```diff
@@ -1,4 +1,4 @@
-// +build !appengine
+// +build !appengine, go1.14
```

Comment:
You probably don't want the space in the build constraint. See: https://golang.org/cmd/go/#hdr-Build_constraints

Reply:
Thanks for spotting this! Nice catch!